docs: RENOVATE_CONFIG_FILE environment variable (#13350)
renovatebot/renovate: docs/usage/getting-started/running.md

### Global config

> Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters. Renovate autodetects a global config file named `config.js`.

Reviewer: ```suggestion
Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters.
By default Renovate checks if a file named `config.js` if present.
```

Author: In my opinion this misses the point of documenting the RENOVATE_CONFIG_FILE variable.

Reviewer: I'm confused:
1. My suggestion is to add a new line, according to our markdown convention
2. There's still the sentence below

Author: Ahh got you. For some reason I was not able to commit GitHub review suggestions on the spot 😄

### Global config

Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters.

> By default Renovate checks if a file named `config.js` if present.

Reviewer: ```suggestion
By default Renovate checks if a file named `config.js` is present.
```

### Global config

> Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters. Renovate autodetects a global config file named `config.js`. Any other (`*.js`, `*.json`, `*.json5`, `*.yaml` or `*.yml`) file is supported, when you reference it with the `RENOVATE_CONFIG_FILE` environment variable (e.g. `RENOVATE_CONFIG_FILE=config.yaml`).

Reviewer: ```suggestion
Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters. Renovate autodetects a global config file named `config.js`.
Any other (`*.js`, `*.json`, `*.json5`, `*.yaml` or `*.yml`) file is supported, when you reference it with the `RENOVATE_CONFIG_FILE` environment variable (e.g. `RENOVATE_CONFIG_FILE=config.yaml`).
```

### Global config

> Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file (`config.js`, `config.json`, `config.json5`, `config.yaml` or `config.yml`), environment variables, or CLI parameters.

Reviewer: I think that this line above is perhaps misleading. By default we only support `config.js`, but we can now support any file of any name with `.json`, `.json5`, `.yaml` or `.yml` file type

Author: Yes, I agree. I updated the paragraph. I believe the any `*.js` named file is also supported with the `RENOVATE_CONFIG_FILE` variable set?

Reviewer 2: Yes
